.shopify-build/polaris-system-integration-test.yml

```diff
@@ -0,0 +1,29 @@
+containers:
+  default:
+    docker: gcr.io/shopify-docker-images/apps/production/web-platform-ci-node:10.18.0-yarn-1.21.1
+  styleguide:
+    docker: gcr.io/shopify-docker-images/apps/production/web-platform-ci-node:10.13.0-yarn-1.21.1
+
+shared:
+  build-polaris: &build-polaris
+    label: 'Build polaris'
+    container: default
+    timeout: 35m
```

Comment:
`35m` seems long, is this what they use in web?

Reply:
Nope it's from when I was debugging and didn't want the container to close on me while I was SSH'd in, we can easily make this 5m :D